### ISSUE TITLE
Using My Calendar with Twenty Twenty Three (FSE theme) - Event content information not included in the Post Content block.

### ISSUE THREAD

**Issue by paaljoachim** [joedolson/my-calendar#508]
**Describe the bug**
I am exploring how My Calendar works in Twenty Twenty Three. (The Gutenberg plugin is not active. Only Query Monitor and My Calendar is active.)
From what I see the Post Content block does not show the post content of My Calendar.

**To Reproduce**
1. Activate Twenty Twenty Three.
2. Create a few events that contain. A featured image. Post content text and an image.

Example. Backend default event.
Contains a Title and post content (text and image).
![Screenshot 2022-12-22 at 00 23 04](https://user-images.githubusercontent.com/5323259/209063021-82d315a9-c8e7-43cd-93fd-3b45ab8665ae.jpg)

Contains a featured image.
![Screenshot 2022-12-22 at 00 23 46](https://user-images.githubusercontent.com/5323259/209063085-4ae48fed-81b1-4b6f-bac5-028697744f21.jpg)

4. View the calendar on the frontend. Click an event. A popup is seen with title, date, featured image, excerpt, location, map and Read more.

Zoomed out view:
![Screenshot 2022-12-22 at 00 26 32](https://user-images.githubusercontent.com/5323259/209063423-ade1ce52-3512-4043-82bb-56df58461525.jpg)

5. Click Read more. 

Example. (Showing the Single template.)

![Screenshot 2022-12-22 at 00 27 54](https://user-images.githubusercontent.com/5323259/209063602-726f8d13-ba85-4705-8bb7-c8b4a1983186.jpg)
Featured image is seen.
Post/Event title is seen.
The Post content is missing from the above screenshot.

7. Going to Site Editing. Opening Single template.
![Screenshot 2022-12-22 at 00 29 32](https://user-images.githubusercontent.com/5323259/209063807-d526f371-6768-4f92-bdad-0f3d53867ed3.jpg)

The Post Content block is seen in the Single Template.

**Expected behavior**
I expected to see event content information inside the Post Content block.

**Comment by paaljoachim** [comment]
I went to Settings -> Display
Main View (Popup or List)
Turned on Author, Host and Description.

![bluewinddesign com_my-calendar_](https://user-images.githubusercontent.com/5323259/209066503-8fd68bce-6d8a-4f95-ab68-545280aa3fc4.jpg)

It seems description = post content.

I clicked Read more to move on to the Single event content.
---

In Settings -> Display.
Single Event. I turned on everything except Google Map.

Same as earlier post content is not seen in Single event view.
